CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+Contributing to Jellyfin
+============
+
+Thank you for your interest in contributing to Jellyfin. We are a community project made up of volunteers doing our best to keep a free software version of Emby alive and healthy.
+
+This document describes how to contribute to Jellyfin via GitHub, and is applicable to anyone who might wish to open up an issue or do a pull request. Please give it a read-through first.
+
+## Issues
+
+Issues should be one of three things:
+
+1. A software bug with Jellyfin.
+2. A feature request or suggestion.
+3. A task to be done with Jellyfin.
+
+All other discussions should be directed towards the [Jellyfin Riot channel](https://matrix.to/#/#jellyfin:matrix.org), the [Jellyfish development Riot channel](https://matrix.to/#/#jellyfin-dev:matrix.org), or [our subreddit](https://reddit.com/r/jellyfin).
+
+When writing an issue, please ensure you capture as much relevant detail as possible - having to request additional information just leads to wasted time. Please include any log output, configuration options, and system details.
+
+## Pull Requests
+
+We adhere to a fairly standard fork-and-PR model of development:
+
+1. Fork a copy of the Jellyfin repository to your own user.
+
+2. Make your changes on a local feature branch of your copy of the repository.
+
+3. Submit a pull request from your feature branch back to the upstream master branch.
+
+All PRs require review before merging. Once that is done, if the PR is not tagged with the WIP label, it should be immediately merged by the reviewer assuming that they have write access. If you do not have write access, please eel free to leave reviews as well and suggest changes as they will be taken into account as well.
+
+## Developer Access
+
+We are open with granting developer (merge/write) access to those who show interest in contributing and reviewing PRs. Such access can be granted by a current developer. Unruly or abusive developers can be removed at any time by the core team, so please take your responsibility seriously. We're all in this together trying to make the best free software media platform we can.
+
+## How can I contribute?
+
+If you're a .NET C# or Javascript developer, please feel free to jump right. If you're not a developer, please consider helping out by contributing documentation, testing the latest releases and reporting bugs, or by telling your friends about the project! Any little bit helps.
```

Comment:
"jump right" to "jump right in".

Move the "If you're not a developer..." to a new line?

Reply:
Fixed and added some more words.